internal/distro/fedora31/distro.go

```diff
@@ -149,6 +149,7 @@ func New() *Fedora31 {
 			"qemu-img",
 			"systemd",
 			"tar",
+			"xz",
```

Comment:
Hm, how come these tests passed before your PR? Something sounds off...

Reply:
I agree, but I haven't yet come with a good theory why is that.

Reply:
It's because `x86_64` has:
```
				BuildPackages: []string{
					"grub2-pc",
				},
```
where `grub2-pc` requires `dracut` which in turn requires `xz`. `aarch64` does not require anything that requires `xz` therefore the error.

Reply:
Thanks!